web/pandas/pdeps/0014-string-dtype.md

```diff
@@ -215,16 +215,16 @@ and NaN semantics.
 Overview of the different ways to specify a dtype and the resulting concrete
 dtype of the data:
 
-| User specification                     | Concrete dtype                                    | String alias            | Note |
-|----------------------------------------|---------------------------------------------------|-------------------------|------|
-| Unspecified (inference)                | `StringDtype(storage="pyarrow"|"python", na_value=np.nan)` | "string"                | (1)  |
-| `StringDtype()` or `"string"`              | `StringDtype(storage="pyarrow"|"python", na_value=np.nan)` | "string"                | (1), (2)  |
-| `StringDtype("pyarrow")`                 | `StringDtype(storage="pyarrow", na_value=np.nan)`           | "string"                | (2)     |
-| `StringDtype("python")`                  | `StringDtype(storage="python", na_value=np.nan)`            | "string"                | (2)     |
-| `StringDtype("pyarrow", na_value=pd.NA)` | `StringDtype(storage="pyarrow", na_value=pd.NA)`            | "string[pyarrow]"       |      |
-| `StringDtype("python", na_value=pd.NA)`  | `StringDtype(storage="python", na_value=pd.NA)`            | "string[python]"        |      |
-| `StringDtype(na_value=pd.NA)`            | `StringDtype(storage="pyarrow"|"python", na_value=pd.NA)` | "string[pyarrow]" or "string[python]" | (1)  |
-| `StringDtype("pyarrow_numpy")`           | `StringDtype(storage="pyarrow", na_value=np.nan)`           | "string[pyarrow_numpy]" | (3)  |
+| User specification                       | Concrete dtype                                                | String alias                          | Note     |
+|------------------------------------------|---------------------------------------------------------------|---------------------------------------|----------|
+| Unspecified (inference)                  | `StringDtype(storage="pyarrow"\|"python", na_value=np.nan)`   | "string"                              | (1)      |
+| `StringDtype()` or `"string"`            | `StringDtype(storage="pyarrow" \| "python", na_value=np.nan)` | "string"                              | (1), (2) |
```

Comment:
The problem is that for our online web version, the `\` just renders as well and doesn't seem to work as escaping the `|`. 
Now, short term people are mostly looking at this through the github diff, so maybe this is worth it, and I should just remember to switch it back before merging ..

Reply:
Ah, I see. No problem if you just want to close this then. Reading with the \ in there is still very manageable on the webpage; whereas it is not readable on GitHub. But this is quite minor regardless.

Reply:
Yes, that is a good reason for now to merge this!